### ISSUE TITLE
AM sometimes crashes when quickly opening and closing app details

### ISSUE THREAD

**Issue by Atrate** [MuntashirAkon/AppManager#224]
**Describe the bug**
When quickly going into an app's details and immediately exiting, AM sometimes crashes.

**To Reproduce**
Steps to reproduce the behaviour:
1. Click on an app in AM
2. Exit with the back button
3. Click again
4. Repeat until AM crashes

**Expected behavior**
AM should not crash.

**Crash logs**
```
java.lang.IllegalStateException: onGetLayoutInflater() cannot be executed until the Fragment is attached to the FragmentManager.
    at androidx.fragment.app.Fragment.getLayoutInflater(Fragment.java:1658)
    at androidx.fragment.app.Fragment.onGetLayoutInflater(Fragment.java:1607)
    at androidx.fragment.app.Fragment.performGetLayoutInflater(Fragment.java:1639)
    at androidx.fragment.app.Fragment.getLayoutInflater(Fragment.java:1624)
    at io.github.muntashirakon.AppManager.details.AppInfoFragment.addToHorizontalLayout(AppInfoFragment.java:1094)
    at io.github.muntashirakon.AppManager.details.AppInfoFragment.setHorizontalActions(AppInfoFragment.java:549)
    at io.github.muntashirakon.AppManager.details.AppInfoFragment.lambda$wHiltEPNng0WjHzbhrZ6tjdf4-o(Unknown Source:0)
    at io.github.muntashirakon.AppManager.details.-$$Lambda$AppInfoFragment$wHiltEPNng0WjHzbhrZ6tjdf4-o.run(Unknown Source:2)
    at android.os.Handler.handleCallback(Handler.java:883)
    at android.os.Handler.dispatchMessage(Handler.java:100)
    at android.os.Looper.loop(Looper.java:214)
    at android.app.ActivityThread.main(ActivityThread.java:7356)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:915)

Device Info:
App version: 2.5.21-PRE
App version code: 377
Android release version: 10
Android SDK version: 29
Device brand: lge
Device manufacturer: LGE
Device name: lucye
Device model: LG-H870
Device product name: lucye_global_com
Device hardware name: qcom
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]
System language: en-US
In-App Language: auto
```

Mode: root

**Comment by MuntashirAkon** [comment]
Related issue #135.

**Comment by MuntashirAkon** [comment]
Should be fixed after 5610eb77 and 8eb81051.
